media/static-bundles.json

```diff
@@ -1155,6 +1161,20 @@
       ],
       "name": "about-transparency"
     },
+    {
+      "files": [
+        "js/base/mozilla-traffic-cop.js",
+        "js/firefox/new/better-browser-experiment.js"
+      ],
+      "name": "firefox_new_better_browser_experiment"
```

Comment:
Looks like this snuck back in in the rebase.

Reply:
D'oh. There were merge conflicts, I think I resolved this wrong. Fixing.